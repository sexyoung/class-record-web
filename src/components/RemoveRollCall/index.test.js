import { render, screen } from '@testing-library/react';
import UserEvent from "@testing-library/user-event";
import { RemoveRollCall } from '.';

it('should confirm be triggered', () => {
  const confirm = jest.fn();
  const { container, getByRole } = render(<RemoveRollCall confirm={confirm} />);
  UserEvent.click(getByRole('button'));
  expect(confirm.mock.calls.length).toBe(1);
  expect(container).toMatchSnapshot();
});

