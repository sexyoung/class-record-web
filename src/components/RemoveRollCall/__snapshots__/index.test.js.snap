// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should confirm be triggered 1`] = `
<div>
  <div
    class="RemoveRollCall"
  >
    <div
      class="title"
    >
      刪除點名
    </div>
    <div
      class="name"
    >
      確定刪除嗎?
    </div>
    <button
      class="btn w-full mt-3"
    >
      刪除
    </button>
  </div>
</div>
`;
